pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -0,0 +1,262 @@
+package nstemplateset
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/codeready-toolchain/toolchain-common/pkg/condition"
```

Comment:
could you reorganize the imports?

Reply:
Mostly, I will take care at end before merging PR.  So no need to do it multiple time.